.eslintrc.js

```diff
@@ -102,7 +102,7 @@ module.exports = {
     'jsdoc/check-indentation': 'off',
     'no-restricted-exports': 'off',
     'no-use-before-define': 'off',
-    '@typescript-eslint/no-use-before-define': ['error'],
+    '@typescript-eslint/no-use-before-define': ['error', 'nofunc'],
```

Comment:
Can we allow this? It means we can take advantage of function hositing, which is nice in, for example, `CreateUserWizardValidation.ts`, as all the helper functions can be defined at the bottom of the file, leaving the actual validation schemas at the top. 

It's probably only going to be relevant in files like this one, since we don't generally use `function` declarations (which are necessary inside yup tests to be able to access the helpers that yup provides on `this`. )